stonesoup/predictor/kalman.py

```diff
@@ -133,6 +133,34 @@ def _predict_over_interval(self, prior, timestamp):
 
         return predict_over_interval
 
+    def _predicted_covariance(self, prior, predict_over_interval, **kwargs):
```

Comment:
I feel like the name ``_get_predicted_covariance()`` seems more sensible, seeing as we are not extracting a property of the class (or the underlying models), but rather performing an computation. At the same time it makes the name quite lengthy so happy to leave as is.

Reply:
I thought class.get_something() was considered somewhat non-pythonic, but I may be confused about why that is (something to do with properties). And I prefer the shorter names.

But maybe you want to discourage someone doing `class._predicted_covariance = AMatrix`.

Happy to receive guidance on getters/setters.

Reply:
> But maybe you want to discourage someone doing `class._predicted_covariance = AMatrix`.

Correct. Personally, when I seen these namings my head thinks it's an attribute. But maybe that's just me.

In the case that ``_get_smth()`` is problematic, ``_calc_smth()`` could be used instead but this makes the names even longer.

Reply:
I think @sglvladi makes a good point, and name does look more like a property, than a method. I think _calculate_ is better than _get_ to make it clear your calculating something, rather than just getting something. I think it's better to have clearer names, than worry about the length.

Reply:
Can we leave this for now? At least until we've agreed a 'corporate policy'. This argument also applies to `._transition_matrix()` in the current class, but could easily extend to stuff like the `.matrix()` or `.covar()` methods in the general models, neither of which are properties but return things that look like them. It feels like we should think of a consistent approach before I set off applying a smattering of semi-consistency.